core/analysis/segmentation_token_stream.cpp

```diff
@@ -20,20 +20,24 @@
 /// @author Andrei Lobov
 ////////////////////////////////////////////////////////////////////////////////
 
+#include "segmentation_token_stream.hpp"
+
+#include <frozen/unordered_map.h>
+
 #include "unicorn/segment.hpp"
 #include "unicorn/string.hpp"
-#include <frozen/unordered_map.h>
 #include "velocypack/Slice.h"
 #include "velocypack/Builder.h"
 #include "velocypack/Parser.h"
-
+#include "velocypack/velocypack-aliases.h"
+#include "utils/vpack_utils.hpp"
 #include "utils/hash_utils.hpp"
-#include "segmentation_token_stream.hpp"
+
 
 namespace {
 
-constexpr irs::string_ref CASE_CONVERT_PARAM_NAME  = "case";
-constexpr irs::string_ref BREAK_PARAM_NAME         = "break";
+constexpr VPackStringRef CASE_CONVERT_PARAM_NAME  = VPackStringRef("case");
+constexpr VPackStringRef BREAK_PARAM_NAME         = VPackStringRef("break");
 
 const frozen::unordered_map<
     irs::string_ref,
```

Comment:
We could move to velocypack::StringRef to avoid conversions later

Reply:
postponed